Examples/BookingSystem.AspNetCore/Program.cs

```diff
@@ -19,6 +14,11 @@ public static void Main(string[] args)
 
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(builder =>
+                {
+                    var environmentName = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+                    builder.AddJsonFile($"appsettings.{environmentName}.json", false, true);
+                })
```

Comment:
It turns out these lines aren't necessary, as it picks up the env specific files automatically